src/main/java/Sunny.java

```diff
@@ -0,0 +1,109 @@
+import java.util.Scanner;
+
+public class Sunny {
+    public static void main(String[] args) {
+        String chatBotName = "Sunny";
+        Task[] tasks = new Task[100]; //Fixed-size array to store tasks
+        int counter = 0; //Counter to keep track of the number of tasks
+
+        //Greets the user
+        System.out.println("Hello! I'm " + chatBotName);
+        System.out.println("What can I do for you?");
+        System.out.println(" ");
+
+        //Initialises scanner for user input
+        Scanner scanner = new Scanner(System.in);
+
+        //Echoes commands after user says bye
+        while (true) {
+            //Gets user input
+            String command = scanner.nextLine();
+
+            //If the user input is "bye"
+            if (command.equalsIgnoreCase("bye")){
```

Comment:
Add a space after before the opening brace. 
e.g "if (condition) {" instead of  "if (condition){"
same for other if else and else if statements